src/Stratis.Bitcoin.Features.Miner/Controllers/StakingApiController.cs

```diff
@@ -63,9 +59,6 @@ public StakingApiController(IFullNode fullNode, ILoggerFactory loggerFactory, IW
         [HttpGet]
         public IActionResult GetStakingInfo()
         {
-            if (this.fullNode.Network.IsBitcoin())
```

Comment:
Ah dude,  this need to be put back, your test is failing cause you just need to set the network on the mocked FullNode object.

Reply:
This was the code that I put in originally to return out of the Staking API methods on the Bitcoin
network.  

I have reverted this change and we have refactored the controllers so the feature setup for BitcoinD and StratisD will use the relevant controllers..